docs/model-types/README.md

```diff
@@ -2,6 +2,7 @@
 
 By leveraging existing third-party model servers, we support a number of standard ML model formats out-of-the box, with more to follow. Currently supported model types:
 
+- [keras] (keras.md)
 - [LightGBM](lightgbm.md)
```

Comment:
Maybe remove the blank character between "]" and "("?

Reply:
Right, also, a nit: capitalize the K in keras.